src/common/context/forms/index.tsx

```diff
@@ -64,25 +65,34 @@ export const FormsContextProvider: FunctionComponent = ({ children }) => {
     setActiveFormIndex(newIndex);
   };
 
-  const newPopulatedForm = (templateIndex: number, data: Array<FormEntry>): void => {
-    const newFormTemplate = config?.forms[templateIndex];
-    const newFormName = newFormTemplate?.name ?? "Document";
-    const formEntries: FormEntry[] = [];
-    const extension = config?.forms[templateIndex]?.extension ?? "tt";
-    for (let index = 0; index < data.length; index++) {
-      formEntries.push({
-        templateIndex,
-        data: {
-          formData: data[index],
-          schema: newFormTemplate?.schema,
-        },
-        fileName: data[index].fileName ?? `${newFormName}-${forms.length + 1 + index}`,
-        ownership: data[index].ownership ?? { beneficiaryAddress: "", holderAddress: "" },
-        extension: extension,
-      });
+  const newPopulatedForm = (templateIndex: number, data: Array<FormEntry>, fileName?: string): void => {
+    try {
+      const newFormTemplate = config?.forms[templateIndex];
+      const newFormName = newFormTemplate?.name ?? "Document";
```

Comment:
I realise something, if the current config.form[0].name = "form name", the file name will be "form name-1",
able to update this logic so that it will replace the space with a "-"? i think it's a bug from previous.